// @hubspot/api-client/lib/codegen/crm/lists/models/PublicListConversionInactivity.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.PublicListConversionInactivityTimeUnitEnum = exports.PublicListConversionInactivityConversionTypeEnum = exports.PublicListConversionInactivity = void 0;
class PublicListConversionInactivity {
    static getAttributeTypeMap() {
        return PublicListConversionInactivity.attributeTypeMap;
    }
    constructor() {
    }
}
exports.PublicListConversionInactivity = PublicListConversionInactivity;
PublicListConversionInactivity.discriminator = undefined;
PublicListConversionInactivity.mapping = undefined;
PublicListConversionInactivity.attributeTypeMap = [
    {
        "name": "offset",
        "baseName": "offset",
        "type": "number",
        "format": "int32"
    },
    {
        "name": "conversionType",
        "baseName": "conversionType",
        "type": "PublicListConversionInactivityConversionTypeEnum",
        "format": ""
    },
    {
        "name": "timeUnit",
        "baseName": "timeUnit",
        "type": "PublicListConversionInactivityTimeUnitEnum",
        "format": ""
    }
];
var PublicListConversionInactivityConversionTypeEnum;
(function (PublicListConversionInactivityConversionTypeEnum) {
    PublicListConversionInactivityConversionTypeEnum["Inactivity"] = "INACTIVITY";
})(PublicListConversionInactivityConversionTypeEnum = exports.PublicListConversionInactivityConversionTypeEnum || (exports.PublicListConversionInactivityConversionTypeEnum = {}));
var PublicListConversionInactivityTimeUnitEnum;
(function (PublicListConversionInactivityTimeUnitEnum) {
    PublicListConversionInactivityTimeUnitEnum["Day"] = "DAY";
    PublicListConversionInactivityTimeUnitEnum["Week"] = "WEEK";
    PublicListConversionInactivityTimeUnitEnum["Month"] = "MONTH";
})(PublicListConversionInactivityTimeUnitEnum = exports.PublicListConversionInactivityTimeUnitEnum || (exports.PublicListConversionInactivityTimeUnitEnum = {}));
